Src/ILGPU.Algorithms.Tests/HistogramTests.tt

```diff
@@ -79,15 +84,36 @@ namespace ILGPU.Algorithms.Tests
 
         #region Histogram Modulo Operations
 
-<# foreach (var inputType in inputTypes) { #>
+<# foreach
+
+    (var inputType in inputTypes) { #>
         internal readonly struct ModuloBin<#= inputType.Name #>Operation
             : IComputeSingleBinOperation<<#= inputType.Type #>, Index1D>
         {
             public readonly Index1D ComputeHistogramBin(
                 <#= inputType.Type #> value,
                 Index1D numBins) =>
-<#      if (inputType.IsFloat) { #>
+<#
```

Comment:
Same here: I think we should restrict uses of these types to specialized TensorCore operations rather than pushing for general purpose compute at the moment. Otherwise, this can have unintended side effects in terms of getting unexpected slow downs.
@MoFtZ @NullandKale